app/assets/javascripts/backbone/views/form_items_view.js.coffee

```diff
@@ -0,0 +1,93 @@
+class ELMO.Views.FormItemsView extends Backbone.View
+
+  el: '.form-items'
+
+  events:
+    'click .add-group': 'show_new_group_modal'
+    'click .form-group .edit': 'show_edit_group_modal'
+    'click .form-group .delete': 'delete_group'
+
+  initialize: (params) ->
+    this.nested_list()
+    this.params = params
+
+  show_new_group_modal: (link) ->
+    link.preventDefault()
+    ELMO.app.loading(true)
+
+    $.ajax({
+      url: ELMO.app.url_builder.build('qing-groups', 'new'),
+      method: "get",
+      data: {form_id: this.params.form_id},
+      success: (html) =>
+        new ELMO.Views.GroupModalView({html: html, list_view: this, mode: 'new'})
+        ELMO.app.loading(false)
+    })
+
+  show_edit_group_modal: (link) ->
+    link.preventDefault()
+    $link = $(link.currentTarget)
+    @form_item_being_edited = $link.closest('.form-item')
```

Comment:
Line 21: In this spot, the data being passed is the form id, so that a new group can be created. It needs that parameter. Is that the line you are referring to?